actions/helm_stage.go

```diff
@@ -32,6 +32,15 @@ func helmStage(ghClient *github.Client, c *cli.Context, helmChart helmChart) {
 		log.Fatalf("Error creating dir %s (%s)", filepath.Join(stagingDir, "tpl"), err)
 	}
 
+	// stage 'tpl/generate_params.toml' with latest git shas for each component
+	defaultParamsComponentAttrs := genParamsComponentAttrs{
+		Org:        c.GlobalString(OrgFlag),
+		PullPolicy: c.GlobalString(PullPolicyFlag),
+		Tag:        c.GlobalString(TagFlag),
+	}
+	paramsComponentMap := getParamsComponentMap(ghClient, defaultParamsComponentAttrs, helmChart.Template, c.GlobalString(RefFlag))
+	generateParams(ourFS, stagingDir, paramsComponentMap, helmChart)
+
```

Comment:
@arschles I updated this PR with this order change.  Since we now track `generate_params.toml` for e2e, we need to first generate it and then run `updateFilesWithRelease`.  

Again, the hope is that this logic will be removed/refactored in deisrel#11